pkg/proxy/ipvs/proxier.go

```diff
@@ -1415,6 +1420,34 @@ func (proxier *Proxier) syncProxyRules() {
 	proxier.deleteEndpointConnections(endpointUpdateResult.StaleEndpoints)
 }
 
+func (proxier *Proxier) acceptIPVSTraffic() {
+	sets := []*IPSet{proxier.clusterIPSet, proxier.externalIPSet, proxier.lbIngressSet, proxier.nodePortSetTCP}
```

Comment:
I'm wondering if it might be better to combine these 4 sets into one, maybe using the list:set IPSet type. That way we only need 2 new rules instead of 5. I'm not sure if there is a significant performance benefit for this or if it is worth it for the benefit of having slightly less rules. What do you think?

Reply:
@rramkumar1 .My mistake, I guess we don't have to ACCEPT nodeport related traffic.
If so, we have only have `clusterIPset/externalIPset/lbIngressSet` need to accept, so up to three iptables rules.
If we add a new ipset, then we need another set to maintain. Also, I don't know I'm not sure list:set type ipset performance, but I don't think it have better performance.

Anyway, I think if we only need three extra rules the most, then we don't have to introduce a new set.

Reply:
Sounds good. Go ahead and make that change and I'll give you an LGTM  :)

Reply:
Fixed, thanks